hooks/build

```diff
@@ -5,6 +5,6 @@ GROUPID=1003
 
 echo "------ DOCKER HUB BUILD HOOK - START -------"
 
-docker build --build-arg BUILD_DATE=`date -u +"%Y-%m-%d %H:%M:%S"` --build-arg userid=$USERID --build-arg groupid=$GROUPID -t $IMAGE_NAME .
+docker build --build-arg userid=$USERID --build-arg groupid=$GROUPID -t $IMAGE_NAME .
```

Comment:
What is going on here?

Reply:
It is my previous change which is not deserves for separate commit. It redefines build command for dockerhub and I forgot to clean up one extra argument

Reply:
If it is not related with other changes, it should be a separate commit (and e.g. mention clearly in the commit message that you forgot it in <hash of original commit>). You can, if you want to, take it in this PR to skip the overhead of creating another PR. Just split it out in a separate commit.